src/client/styles/components/_profile-banner.scss

```diff
@@ -19,6 +19,7 @@
       position: absolute;
       height: 60px;
       top: calc(14vh - 30px);
+      width: 100vw;
```

Comment:
actually all the width should be `100%` instead of `100vw` in case there are extra stuff in the viewport